src/components/presentational/ProgressBar/ProgressBar.css

```diff
@@ -0,0 +1,51 @@
+.mdhui-progress-bar {
+	display: block;
+	width: 100%;
+	position: relative;
+	--icon_size: 24px;
+	--icon_border_width: 2px;
+}
+
+.mdhui-progress-bar .mdhui-progress-bar-default-margin {
+	margin: 16px;
+}
+
+.mdhui-progress-bar .mdhui-progress-bar-background {
+	border-radius: 500px;
+	border-width: var(--icon_border_width) ;
```

Comment:
Huh...did you just add a border to this?  I didn't think there was a border before

Reply:
i would probably drop the border because it makes it look a bit awkward when stuff overlaps it:
<img width="196" alt="Screenshot 2023-11-30 at 2 04 36 PM" src="https://github.com/CareEvolution/MyDataHelpsUI/assets/1643171/67594572-ca71-4fa7-aeaf-10a3a313fd35">